misc/macros.json

```diff
@@ -25,6 +25,8 @@
     "port_in_usage_perc": "((%ports.ifInOctets_rate*8) \/ %ports.ifSpeed)*100",
     "port_out_usage_perc": "((%ports.ifOutOctets_rate*8) \/ %ports.ifSpeed)*100",
     "port_now_down": "%ports.ifOperStatus != %ports.ifOperStatus_prev && %ports.ifOperStatus_prev = \"up\" && %ports.ifAdminStatus = \"up\" && %macros.port",
+    "port_has_xdp_neighbours": "(%macros.port && %links.local_port_id = %ports.port_id)",
+    "port_has_xdp_neighbours_device": "(%macros.port_has_neighbours && (%links.remote_port_id IS NOT NULL))",
```

Comment:
This doesn't seem to work and I think it's because you reference another macro which then references another macro.

Reply:
It does not work because I should not commit before going to bed and just after renaming a macro :D With all testing done before renaming ... 

Anyway, With the correct name, it works beautifully.